aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -76,7 +85,7 @@ std::array<SDPBackend, num_backends> priority_order(sdp_params const& params) {
       SDPBackend::flash_attention,
       SDPBackend::efficient_attention,
       SDPBackend::math};
-  static const bool prefer_cudnn = check_prefer_cudnn_attention();
+  static const bool prefer_cudnn = check_prefer_cudnn_attention(params);
```

Comment:
This is dependent on params so it can't be static anymore or it could be wrong, correct?

Reply:
good catch!